class/defaults.yml

```diff
@@ -15,6 +15,17 @@ parameters:
       tune_fast_device_class: false
       # Rendered into rook-config-override CM
       config_override:
+        global:
+          # Configure full ratios to match the alerts shipped with Rook.
+          # With this config the cluster goes readonly at 85% utilization.
+          # These configs only apply at cluster creation.
+          # To adjust the ratios at run time, use
+          #   `ceph osd set-{nearfull,backfillfull,full}-ratio`
+          # NOTE: we're giving ratios as strings to avoid float rounding
+          # issues when manifesting the values in the resulting config file.
+          mon_osd_full_ratio: 0.9
```

Comment:
```suggestion
          mon_osd_full_ratio: '0.9'
```

Reply:
That's supposed to be '0.85' anyway, I changed it to copy the "broken" value for the docs and accidentally committed it :see_no_evil: